migration-engine/connectors/sql-migration-connector/src/sql_database_migration_inferrer.rs

```diff
@@ -81,11 +97,57 @@ fn infer(
     current_database_schema: SqlSchema,
     expected_database_schema: SqlSchema,
     flavour: &dyn SqlFlavour,
+    next_datamodel: &Datamodel,
 ) -> SqlMigration {
     let steps =
         sql_schema_differ::calculate_steps(Pair::new(&current_database_schema, &expected_database_schema), flavour);
+    let next_schema = &expected_database_schema;
+
+    let added_columns_with_virtual_defaults: Vec<(usize, usize)> = steps
```

Comment:
🙏

Reply:
Maybe split the iterator into pieces and name the pieces?